package echo;

message EchoRequest
{
  required string message = 1;
  required string serviceName = 2;
  required string funName = 3;
};

message EchoResponse
{
  required string response = 1;
};

service EchoService
{
  rpc Echo(EchoRequest) returns (EchoResponse);
};
option cc_generic_services = true;